src/drm/fairplay.js

```diff
@@ -79,7 +82,7 @@ export default class FairPlay extends BaseDrmProtocol {
     let request = new XMLHttpRequest();
     request.responseType = "text";
     request.addEventListener("load", FairPlay._licenseRequestLoaded, false);
-    request.addEventListener("error", FairPlay._licenseRequestFailed, false);
+    request.addEventListener("error", ()=>FairPlay._onError.bind(Error.Code.LICENSE_REQUEST_FAILED), false);
```

Comment:
why do u need bind here? this is both an error function that keeps the `this` context and this is also a call to static method so there's no `this` context